app/assets/javascripts/social_share_button.js

```diff
@@ -1,21 +1,22 @@
-function openShareUrl(url, initialWidth = 640, initialHeight = 480) {
-  const width = Math.max(100, Math.min(screen.width, initialWidth));
-  const height = Math.max(100, Math.min(screen.height, initialHeight));
+(function () {
+  function openShareUrl(url, initialWidth = 640, initialHeight = 480) {
```

Comment:
Can't this one be fixed by just moving the function inside the ready call, rather than adding another layer using the horrid IIFE syntax?

Reply:
Perhaps we can use the following pattern. But we have to skip `no-lone-blocks` rule.

```js
// eslint ignore no-lone-blocks
{
  function FOSSGISOSRMEngine(id, vehicleType) { ... }
  FOSSGISOSRMEngine(...)
}

Reply:
Well for that specific one we can just move the function inside the ready callback surely?

Reply:
oh, you meant the ready call of jQuery. Would be probably another idea. Yes.